### DISCUSSION TITLE
Searching and/or filtering for response bodies

### DISCUSSION THREAD

**Discussion by ttytm** [darrenburns/posting#145] [Ideas]
Especially for big request a search and filter functionality can be a game changer.

A regular vim search `/` would already go a long way. Filtering via jq would be an outstanding feature.

**Comment by darrenburns** [comment]
As a temporary measure, not sure if you're aware (I can't even remember if I've documented this), if you have your `PAGER`/`EDITOR` env vars set you can quickly hop over to an external program using `f3`/`f4` respectively.

There's also `POSTING_PAGER_JSON` - if the response is JSON, then that'll open in whatever program you have assigned. I personally use `POSTING_PAGER_JSON=fx`, so when I press `f3` with a JSON body focused, it opens `fx` (https://fx.wtf).

You can also set these in the `config.yaml` file as `pager`

**Comment by darrenburns** [comment]
(Although I do agree that at least a simple `/` search is needed in Posting itself!)
